RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate RoomInfoView to Typescript

## Proposed changes

Migrate RoomInfoView to Typescript

## Issue(s)	


## How to test or reproduce


- Access RoomInfoView from RoomView clicking on: profile pic, name
- Inside the RoomView click on the header, then click on the card that is showing the avatar of the channel or user, you should be redirected to RoomInfoView

## Screenshots

## Types of changes




## Checklist



## Further comments
